app/client/cli/servicer.go

```diff
@@ -0,0 +1,284 @@
+package cli
+
+import (
+	"context"
+	sha "crypto"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/pocket/rpc"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/crypto"
+)
+
+func init() {
+	rootCmd.AddCommand(NewServicerCommand())
+}
+
+// TECHDEBT: (unittest) unit test the command: e.g. on number of arguments
+func NewServicerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "Servicer",
+		Short:   "Servicer specific commands",
+		Aliases: []string{"servicer"},
+		Args:    cobra.ExactArgs(0),
+	}
+
+	cmds := servicerCommands()
+	applySubcommandOptions(cmds, attachPwdFlagToSubcommands())
+	applySubcommandOptions(cmds, attachKeybaseFlagsToSubcommands())
+
+	cmd.AddCommand(cmds...)
+
+	return cmd
+}
+
+func servicerCommands() []*cobra.Command {
+	cmdDef := actorCmdDef{"Servicer", coreTypes.ActorType_ACTOR_TYPE_SERVICER}
+	cmds := []*cobra.Command{
+		newStakeCmd(cmdDef),
+		newEditStakeCmd(cmdDef),
+		newUnstakeCmd(cmdDef),
+		newUnpauseCmd(cmdDef),
+		{
+			Use:   "Relay <servicer> <application> <relayChainID> <payload>",
```

Comment:
`s/payload/payloadSerializedProto`?

Please rename `payload` so its self explanatory on what we expect

Reply:
Thanks. Fixed.